builtin/aws/alb/releaser.go

```diff
@@ -132,52 +132,66 @@ func (r *Releaser) resourceSecurityGroupCreate(
 
 		vpc = subnetInfo.Subnets[0].VpcId
 	}
-	sGroup, err := utils.CreateSecurityGroup(ctx, sess, fmt.Sprintf("%s-incoming", lbName), vpc, int(port))
-	if err != nil {
-		return err
+
+	if len(r.config.SecurityGroupIDs) >= 0 {
```

Comment:
```suggestion
	if len(r.config.SecurityGroupIDs) > 0 {
```
If it = 0, then there are no security group IDs, right?